docs/v1/milestones.md

```diff
@@ -0,0 +1,55 @@
+# v1 Milestones
+
+**NOTE: THIS DOC IS A WORK IN PROGRESS**
+
+This document captures concensus on the identity and content of milestones.
+
+## Legos-1
+
+Tentative target date: Feb 1, 2017
+
+1.  `servicecatalog/v1alpha1` API complete and functional:
+  1.  `Broker`, `ServiceClass`, `Instance`, `Binding`
+  2.  Integrators should be able to program against this REST API
+2.  API server serves `servicecatalog/v1alpha1`
+3.  Golang client for API in (1) is auto-generated
```

Comment:
Although not explicitly stated, I think that falls under MVP-1 item 3:

> Native-k8s-style CLI experience

i.e. I don't think @pmorie intended that as a goal for the Legos-1 milestone.